electron/launcher/main-process/http-handler.js

```diff
@@ -0,0 +1,96 @@
+const request = require('request-promise');
+
+// POST REQUEST TO REGISTER USER
+const registerUser = (user) => {
+  let options = {
+    method: 'POST',
+    uri: 'http://127.0.0.1:8000/accounts/signup/',
+    header: {'content-type': 'application/x-www-form-urlencoded'},
```

Comment:
pull this out into a config file - `uri`

Reply:
Done. New config.json file created in /main-process